README.md

```
@@ -123,13 +123,49 @@ To install with multiple extras (e.g., Ollama + llama.cpp):
+curl -fsSL https://copaw.agentscope.io/install.bat -o install.bat && install.bat
+```
+
 **Windows (PowerShell):**
 
 ```powershell
 irm https://copaw.agentscope.io/install.ps1 | iex
 ```
 
-Then open a new terminal and run:
+> **Note**: The installer will automatically check the status of uv. If it is not installed, it will attempt to download and configure it automatically. If the automatic installation fails, please follow the on-screen prompts or execute `python -m pip install -U uv`, then rerun the installer.
+
```

markdownlint-cli2 (0.21.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)